.github/workflows/pr-checks.yaml

```
@@ -90,5 +90,5 @@ jobs:
 
       - name: Upload coverage report
         uses: codecov/codecov-action@13ce06bfc6bbe3ecf90edbbf1bc32fe5978ca1d3 # v5.3.1
-        with:
-          token: ${{ secrets.CODECOV_TOKEN }}
+        env:
+          CODECOV_TOKEN: ${{ secrets.CODECOV_TOKEN }}
```

```suggestion
        env:
          CODECOV_TOKEN: ${{ secrets.CODECOV_TOKEN }}
        with:
          files: gradle/aggregation/build/reports/jacoco/testCodeCoverageReport/testCodeCoverageReport.xml
```

See: https://github.com/hiero-ledger/hiero-consensus-node/blob/main/.github/workflows/node-zxc-compile-application-code.yaml#L525

---

Thank you for the clarification here @jjohannes I was un-aware of such decision and that it was even there.
now you mention it makes sense. 

Also thanks for the suggested changes, I will test it out and if it passes, we can finally close this one 😅 👍

---

Changes done ✅